README.md

```diff
@@ -1 +1,37 @@
-# dp-search-builder
\ No newline at end of file
+dp-search-builder
+==================
+
+### Getting started
+
+* Run ```brew install kafka```
```

Comment:
Consider referring to the dp-import docs for common infrastructure setup, and having only app specifics here.

Reply:
👍